services/api-server/src/simcore_service_api_server/utils/client_base.py

```diff
@@ -30,36 +34,82 @@ async def is_responsive(self) -> bool:
             resp.raise_for_status()
             return True
         except (httpx.HTTPStatusError, httpx.RequestError) as err:
-            log.error("%s not responsive: %s", self.service_name, err)
+            _logger.error("%s not responsive: %s", self.service_name, err)
             return False
 
     ping = is_responsive  # alias
 
 
+class _AsyncClientWithCaptures(httpx.AsyncClient):
+    """
+    Adds captures mechanism
+    """
+
+    async def request(self, method: str, url: URLTypes, **kwargs):
+        response: httpx.Response = await super().request(method, url, **kwargs)
+
+        capture_name = f"{method} {url}"
+        _logger.info("Capturing %s ... [might be slow]", capture_name)
+        try:
+            capture_json = get_capture_msg(name=capture_name, response=response)
```

Comment:
Question why not call it `get_captured_msg`?
```suggestion
            captured_json = get_captured_msg(name=capture_name, response=response)
```